src/main/webapp/resources/js/contexts/AnalysisContext.js

```diff
@@ -31,18 +35,19 @@ export const stateMap = {
   COMPLETED: 5
 };
 
-export const isAdmin = window.PAGE.isAdmin;
-
 const initialContext = {
-  analysis: window.PAGE.analysis,
-  analysisName: window.PAGE.analysisName,
-  analysisState: window.PAGE.analysisState,
-  analysisType: window.PAGE.analysisType.type,
-  analysisViewer: window.PAGE.analysisViewer,
-  isCompleted: window.PAGE.analysisState === "COMPLETED",
-  isError: window.PAGE.analysisState.includes("ERROR"),
-  previousState: window.PAGE.previousState,
-  duration: window.PAGE.duration
+  analysis: null,
```

Comment:
Wouldn't these all default to null or false without stating them here?  Since none of these are objects you can just leave them off and add them as needed.

Reply:
If we don't set them then we get a bunch of undefined errors